common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -1,24 +1,66 @@
 ---
 metadata:
-    display_name: Checkboxes
+    display_name: Checkbox
```

Comment:
oh, wait. This change has widespread effects on the Insights guide and on the description of the student answer distribution report in B&R. There may be other affected guides. I'll set up a DOC story...